salt/states/boto_ec2.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+'''
+Manage EC2
+
+.. versionadded:: TBD
```

Comment:
Bah! I need a little version number cheat sheet stuck on the wall in my cubicle :/ Thanks for clarifying.